scripts/test.sh

```diff
@@ -0,0 +1,4 @@
+#'/bin/bash
+
```

Comment:
There is inconsistency between new lines... there should always be 1 new at the end of the file and between logical blocks. In some places you have 3 and in some places you have multiple.